src/module/rules/CombatRules.ts

```diff
@@ -145,6 +163,10 @@ export class CombatRules {
         // Keep base and modification intact, only overwriting the result.
         modifiedDamage.override = {name: 'SR5.Success', value: 0};
         Helpers.calcTotal(modifiedDamage, {min: 0});
+        modifiedDamage.ap.override = {name: 'SR5.Success', value: 0};
+        Helpers.calcTotal(modifiedDamage.ap);
+        modifiedDamage.type.value = '';
+        modifiedDamage.element.value = '';
```

Comment:
I should consider not clearing the element in cases where the vehicle is hit but the attack is blocked by vehicle armor (as the elemental effect would still apply)